snuba/web/query.py

```diff
@@ -287,10 +295,8 @@ def _format_storage_query_and_run(
     """
     Formats the Storage Query and pass it to the DB specific code for execution.
     """
-    from_clause = clickhouse_query.get_from_clause()
     visitor = TablesCollector()
-    visitor.visit(from_clause)
-    table_names = ",".join(sorted(visitor.get_tables()))
+    table_names = _get_clickhouse_tables(clickhouse_query, visitor)
```

Comment:
How come we need to make these changes to the query pipeline?

Reply:
Oops this was part of a first attempt at getting more data into stats in dry run pipeline, forgot to undo.